src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+//#define IGNORE_TRIVIA
```

Comment:
Same pattern used in http://source.roslyn.io/#Roslyn.Compilers.CSharp.Syntax.UnitTests/Parsing/ParsingTests.cs,3

I propose to keep this to ease behavior testing.

Reply:
we've moved away from this in feature testing.  It's valuable to have the trivia actuall verified.

Reply:
I meant we keep it as-is (commented out), comes in handy once you just started a feature.

Reply:
Ah gotcha!  That's fine with me!